solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,156 @@
+"""Simulating an ATM. """
+
+import sys
+import json
+
+ATM_FILE = 1  # The number 1 refers to the place of the file as a parameter.
```

Comment:
Usually your code need to be simple enough that you don't need doc to explain some line.
Sometime its ok to explain if there is complex logic section, however, variable naming should always be clear without doc